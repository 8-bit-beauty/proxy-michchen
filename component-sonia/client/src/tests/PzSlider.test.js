// import React from 'react';
// import { shallow } from 'enzyme';
// import { shallowToJson } from 'enzyme-to-json';
// import { render } from 'react-dom';
// import PzSlider from '../PzSlider.jsx';

// jest.mock('react-dom');
// const items = {
//   id: 4, productName: 'Sleek Fresh Tuna', productDescription: 'Enim odio vitae. Vel dignissimos omnis cum aut quibusdam.', color: 'yellow', price: 231, imageURL: 'https://s3-us-west-1.amazonaws.com/pictures-hrsf/download+(7).jpeg', rating: 4, reviewNumber: 186, isPrime: 1,
// };

// describe('PzSlider', () => {
//   it('should render correctly in "debug" mode', () => {
//     const component = shallow(<PzSlider debug props={items} />);
//     expect(component).toMatchSnapshot();
//   });
//   it('should render correctly', () => {
//     expect(render).toHaveBeenCalledWith(
//       <PzSlider props={items} />, 'element-node',
//     );
//     expect(render).toHaveBeenCalledTimes(1);
//   });
//   it('should render correctly with no props', () => {
//     const component = shallow(<PzSlider props={items} />);
//     expect(shallowToJson(component)).toMatchSnapshot();
//   });
//   it('renders PzElement component', () => {
//     const wrapper = shallow(
//       <PzSlider item={items} />,
//     );
//     expect(wrapper).toMatchSnapshot();
//   });
// });
