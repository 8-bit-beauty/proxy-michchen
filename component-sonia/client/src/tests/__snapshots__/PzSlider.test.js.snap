// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PzSlider renders PzElement component 1`] = `
<div>
  <h3>
    Customers who bought this item also bought
  </h3>
  <div>
    <div>
      <PzElement
        element={
          Object {
            "color": "yellow",
            "id": 4,
            "imageURL": "https://s3-us-west-1.amazonaws.com/pictures-hrsf/download+(7).jpeg",
            "isPrime": 1,
            "price": 231,
            "productDescription": "Enim odio vitae. Vel dignissimos omnis cum aut quibusdam.",
            "productName": "Sleek Fresh Tuna",
            "rating": 4,
            "reviewNumber": 186,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`PzSlider should render correctly in "debug" mode 1`] = `
<div>
  <h3>
    Customers who bought this item also bought
  </h3>
  <div>
    <div>
      <PzElement
        element={
          Object {
            "color": "yellow",
            "id": 4,
            "imageURL": "https://s3-us-west-1.amazonaws.com/pictures-hrsf/download+(7).jpeg",
            "isPrime": 1,
            "price": 231,
            "productDescription": "Enim odio vitae. Vel dignissimos omnis cum aut quibusdam.",
            "productName": "Sleek Fresh Tuna",
            "rating": 4,
            "reviewNumber": 186,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`PzSlider should render correctly with no props 1`] = `
<div>
  <h3>
    Customers who bought this item also bought
  </h3>
  <div>
    <div>
      <PzElement
        element={
          Object {
            "color": "yellow",
            "id": 4,
            "imageURL": "https://s3-us-west-1.amazonaws.com/pictures-hrsf/download+(7).jpeg",
            "isPrime": 1,
            "price": 231,
            "productDescription": "Enim odio vitae. Vel dignissimos omnis cum aut quibusdam.",
            "productName": "Sleek Fresh Tuna",
            "rating": 4,
            "reviewNumber": 186,
          }
        }
      />
    </div>
  </div>
</div>
`;
